src/middlewares/container.js

```diff
@@ -2,7 +2,7 @@ const container = (ctrl) => async (req, res, next) => {
   try {
     await ctrl(req, res);
   } catch (error) {
-    next(error);
+	res.json({ message: error.message });
```

Comment:
Previous solution was better, because when request fails, it should return error with 500 status code. In current implementation it returns 200 one.

Reply:
but with this solution when we try container with post request it doesn`t work

Reply:
What type of problem do we have with previous solution?

Reply:
Server don`t return nothing, empty message and error, if change this, all is good

Reply:
I probably understand what do you mean. So, in this case, we should manually specify response status code and its data. Should be something like `res.status(500).send({ message: error.message })`